src/services/shims.ts

```diff
@@ -274,10 +274,7 @@ module ts {
         }
 
         public getDefaultLibFileName(options: CompilerOptions): string {
-            // Shim the API changes for 1.5 release. This should be removed once
-            // TypeScript 1.5 has shipped.
-            return "";
-            //return this.shimHost.getDefaultLibFileName(JSON.stringify(options));
+            return this.shimHost.getDefaultLibFileName(JSON.stringify(options));

```

Comment:
you still need the comment, and you still need to shim it.

the problem is that the released 1.4 VS plugin do not have a function called "getDefaultLibFileName", it is called "getDefaultLibFilename" with lowercase "n". this if you use the latest from master on an existing 1.4 installation it will fail. this affects users who want to try the latest bits from master. 

so, to work around this, you need to shim it.. something like:

```TypeScript
if (typeof this.shimHost.getDefaultLibFileName === "function") { 
   return this.shimHost.getDefaultLibFileName(JSON.stringify(options));
}
return "";
```

and i would give it a quick try in the shipped 1.4 VS as well.